docs/tutorial/t-setup-environment.md

```diff
@@ -5,25 +5,28 @@ This is part of the [Charmed Kafka Tutorial](/t/charmed-kafka-k8s-documentation-
 [Multipass](https://multipass.run/) is a quick and easy way to launch virtual machines running Ubuntu. It uses "[cloud-init](https://cloud-init.io/)" standard to install and configure all the necessary parts automatically.
 
 Let's install Multipass from [Snap](https://snapcraft.io/multipass) and launch a new VM using "[charm-dev](https://github.com/canonical/multipass-blueprints/blob/main/v1/charm-dev.yaml)" cloud-init config:
+
 ```shell
 sudo snap install multipass && \
 multipass launch --cpus 4 --memory 8G --disk 50G --name my-vm charm-dev
 ```
-*Note: all 'multipass launch' params are [described here](https://multipass.run/docs/launch-command)*.
+*Note: See all `multipass launch` parameters in the [launch command documentation](https://multipass.run/docs/launch-command)*.
 
 Multipass [list of commands](https://multipass.run/docs/multipass-cli-commands) is short and self-explanatory, e.g. show all running VMs:
+
 ```shell
 multipass list
 ```
 
 As soon as new VM started, enter inside using:
+
 ```shell
 multipass shell my-vm
 ```
-*Note: if at any point you'd like to leave Multipass VM, enter `Ctrl+d` or type `exit`*.
+*Note: if at any point you'd like to leave Multipass VM, use `Ctrl+D` or type `exit`*.
```

Comment:
I would keep it lower case, given it is not `Ctrl+Shift+D`, seems confusing to have it upper case, but I don't know what is the standard for these commands

Reply:
For me, it seems natural to Capitalize the key names.
We have it as Ctrl + C in our [style guide](https://docs.ubuntu.com/styleguide/en#interacting-with-ui-elements) (Keyboard shortcuts and keyboard keys) but no specific instructions on capitalization (I'll add a suggestion to add it).
More examples:
https://en.wikipedia.org/wiki/Control-C
[Microsoft docs - CTRL + Z](https://support.microsoft.com/en-gb/office/change-the-capitalization-or-case-of-text-1d86cf80-fbef-4380-8d6f-59a6b77db749#:~:text=press%20%E2%8C%98%20%2B%20Z%20.-,To%20use%20a%20keyboard%20shortcut%20to%20change%20between%20lowercase%2C%20UPPERCASE,style%20you%20want%20is%20applied.)

Reply:
Sold! :D You are right! TIL! :D